### ISSUE TITLE
I want to change prices of the drinks

### ISSUE THREAD

**Issue by ofosos** [ofosos/strichliste#5]
As an admin of this system, I want to be able to change the price of an item easily.

**Comment by pReya** [comment]
Yes, good idea. Maybe we should also use generic drink items, instead of specific drinks, since the selection of available drinks changes frequently. So maybe just one single item?!
